### PR TITLE
 LPS-178193 Functional Automation |- As a DXP user, I want to have more accessibility configurations available

### PR DESCRIPTION
**Pull request by eltonccdantas**
[Ticket](https://liferay.atlassian.net/browse/LPS-192304)

Create

1. CanEnableReducedMotion
2. CanEnableIncreasedTextSpacing
3. CanEnableExpandedText

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu:nocompile

**Comment by eltonccdantas** [comment]
ci:test:relevant

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu

File: modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro [hunk 1]
```diff
@@ -53,19 +53,19 @@ definition {
...
 		AccessibilityMenu.openAccessibilityMenu();
 
 		Uncheck.uncheckToggleSwitch(
-			key_toggleSwitchLabel = "Underlined Links",
+			key_toggleSwitchLabel = ${key_toggleSwitchLabel},
 			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 	}
 
-	macro toggleUnderlinedLinksOn {
+	macro toggleAnOptionInTheAccessibilityMenuOn {
```
Comment (eltonccdantas) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro:64]: I think I'll change it to `toggleAnOptionOn ` so when we read the `.testcase` it looks more readable `AccessibilityMenu.toggleAnOptionOn`

What do you think?

File: modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro [hunk 2]
```diff
@@ -53,19 +53,19 @@ definition {
 			value1 = "\ENTER");
 	}
 
-	macro toggleUnderlinedLinksOff {
+	macro toggleAnOptionInTheAccessibilityMenuOff {
```
Comment (eltonccdantas) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro:56]: Same as the "On" macro

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu:nocompile

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 3]
```diff
@@ -118,6 +118,66 @@ definition {
 		}
 	}
 
+	@description = "LPS-178192. Verifies Expanded Text effects toggle can be enabled."
+	@priority = 4
+	test CanEnableExpandedText {
+		property portal.acceptance = "true";
+
+		task ("When toggle on reduced motion") {
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:126]: update to "expanded text"

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 4]
```diff
@@ -118,6 +118,66 @@ definition {
...
+			AccessibilityMenu.toggleAnOptionInTheAccessibilityMenuOn(key_toggleSwitchLabel = "Expanded Text");
+		}
+
+		task ("Then A body class “c-prefers-reduced-motion” is applied") {
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:130]: sentence case: `Then a body class`

Comment (john-co) [reply modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:130]: `c-prefers-expanded-text`

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 5]
```diff
@@ -118,6 +118,66 @@ definition {
...
+	@description = "LPS-178192. Verifies Increased Text Spacing effects toggle can be enabled."
+	@priority = 4
+	test CanEnableIncreasedTextSpacing {
+		property portal.acceptance = "true";
+
+		task ("When toggle on reduced motion") {
+			AccessibilityMenu.toggleAnOptionInTheAccessibilityMenuOn(key_toggleSwitchLabel = "Increased Text Spacing");
+		}
+
+		task ("Then A body class “c-prefers-reduced-motion” is applied") {
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:150]: same as previous comment, update the task description

File: modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro (hunk 1 above)
Comment (john-co) [reply modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro:64]: I agree with simplifying the macro name. I suggest: `AccessibilityMenu.enableSpecificOption` for readability.

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 6]
```diff
@@ -86,7 +86,7 @@ definition {
 		task ("And When guest user toggles show underline to OFF") {
 			Navigator.openURL();
 
-			AccessibilityMenu.toggleUnderlinedLinksOff();
+			AccessibilityMenu.toggleAnOptionInTheAccessibilityMenuOff(key_toggleSwitchLabel = "Underlined Links");
 
 			AssertNotChecked.assertNotCheckedNotVisible(
 				key_toggleSwitchLabel = "Underlined Links",
```
[the remark this answers is not in the document]
Comment (john-co) [reply modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:92]: I think it's fine. There are 50 other instances

**Review by john-co — APPROVED** [review]
Please see comments and make adjustments. Besides that looks good to me! :sunglasses:

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:stop:accessibility-menu:nocompile

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu

**Comment by eltonccdantas** [comment]
ci:test:accessibility-menu

**Comment by eltonccdantas** [comment]
ci:test:relevant

File: modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase [hunk 7]
```diff
@@ -546,14 +606,12 @@ definition {
...
 
 		task ("And underline class is applied to body element") {
-			AccessibilityMenu.toggleUnderlinedLinksOn();
+			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
 
 			AssertChecked.assertCheckedNotVisible(
 				key_toggleSwitchLabel = "Underlined Links",
 				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 
-			var key_class = ${key_class};
```
Comment (john-co) [inline modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase:555]: nice

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**